tests/AutoReview/ProjectCodeTest.php

```diff
@@ -357,7 +357,7 @@ public function testThatTestClassCoversAreCorrect(string $testClassName): void
         }
 
         $covers = Preg::match('/@covers (\S*)/', $doc, $matches);
-        static::assertNotFalse($covers, sprintf('Missing @covers in PHPDoc of test class "%s".', $testClassName));
+        static::assertSame(1, $covers, sprintf('Missing @covers in PHPDoc of test class "%s".', $testClassName));
```

Comment:
possibly we can have multiple `@covers` in annotation,
but agree about `notFalse` assertion change

Reply:
So maybe `>=1`? It was reported by PHPStan's extension as wrong assert, so I fixed it (or tried to 😝).

Reply:
do we have a case where we have multiple covers? maybe we should have use matchAll here? `1` is also OK, but it's bit counter-intuitive, one needs to have their eye hangs on it to see we look for first `@covers` only

Reply:
I used `matchAll()` + `assertGreaterThanOrEqual()`.